clang/lib/CIR/CodeGen/CIRGenCall.cpp

```diff
@@ -716,6 +716,19 @@ RValue CIRGenFunction::GetUndefRValue(QualType Ty) {
   return RValue::get(nullptr);
 }
 
+mlir::Value CIRGenFunction::buildRuntimeCall(mlir::Location loc,
+                                             mlir::cir::FuncOp callee,
+                                             ArrayRef<mlir::Value> args) {
+  auto call = builder.create<mlir::cir::CallOp>(loc, callee, args);
+  assert(call->getNumResults() <= 1 &&
+         "runtime functions have at most 1 result");
+
+  if (call->getNumResults() == 0)
+    return nullptr;
+
```

Comment:
Please add a `assert(!UnimplementedFeature::setCallingConv())` here and add a new method in `clang/lib/CIR/CodeGen/UnimplementedFeatureGuarding.h`.  The idea here is to remember to at some point in time handle `call->setCallingConv(getRuntimeCC());`